### DISCUSSION TITLE
Welcome to dbus-mqtt-devices Discussions!

### DISCUSSION THREAD

**Discussion by freakent** [freakent/dbus-mqtt-devices#77] [Announcements]
## Welcome!
  From 1st Feb 2024 this project will use GitHub Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions about functionality or installation problems.
  * Share ideas.
  * Engage with other community members.

Please _do not_ raise Issues without a starting a discussion first. 

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by broekieman** [comment]
Hi,

I am very interested in technical things, so i have Home Assistant and since last week a home battery with Victron inverter. I am using a kwh meter by lan for grid meting. I have solarpanels with Enphase micro inverters who are visible in Home Assistant. Now i would like to send my Enphase production to Victron. Please tell me if this is posible without an extra meter or sensor. I have Mqtt installed in Home Assistant.

**Reply by stundenblume** [reply]
Hi,
if you get the Power of your inverters, you can use this driver here to create a new PV inverter in the victron system and write via MQTT your data to the newly created device.
See this [example](https://github.com/freakent/dbus-mqtt-devices/wiki#get-tasmota-to-act-like-an-victron-recognized-pv-inverter).
Sebastian

**Reply by broekieman** [reply]
How does Tasmota get the data from my Enphase inverters?

**Reply by broekieman** [reply]
Does a shelly 1pm need to be connected to the ac pv? 
Which device can i use from sonoff, does it need a kwh meter build in?

**Reply by stundenblume** [reply]
> How does Tasmota get the data from my Enphase inverters?

That's just an example. You don't need a new (Tasmota) device. You can use the example to generate the same topics via HA using the existing Enphase data.

**Reply by broekieman** [reply]
I have the sensors created in home assistant but no data comes in. It says "unknown"

**Reply by stundenblume** [reply]
On the HA side I can't comment. I use ioBroker. You sad:

> I have solarpanels with Enphase micro inverters who are visible in Home Assistant

So I assume you already have the Enphase data available and just need help to send it to the Victron system.

**Reply by broekieman** [reply]
How can i uninstall this?

**Reply by freakent** [reply]
There is an unistall.sh script in the bin directory. So from the command line just type:
cd /data/driver/dbus-mqtt-devices-<version>
./bin/unistall.sh

**Comment by broekieman** [comment]
Do i need CT clamps or can it read directly from the Enphase gateway?
Met vriendelijke groet,

Maarten Broekman


Op wo 2 okt 2024 om 07:19 schreef Sebastian S. ***@***.***>:
